app/view_models/v1/work_item.rb

```diff
@@ -1,24 +1,60 @@
 module V1
-  class WorkItem < BaseViewModel
+  class WorkItem < BaseWithAdjustments
+    LINKED_TYPE = 'work_items'.freeze
+    ID_FIELDS = %w[id].freeze
+
+    attribute :id, :string
     attribute :work_type, :translated
     # TODO: import time_period code from provider app
     attribute :time_spent
```

Comment:
I would consider converting this to a period so that it is easier to render in the display. :)